js/ui/components/clubhouse.js

```diff
@@ -161,6 +161,10 @@ var ClubhouseComponent = new Lang.Class({
         this._banner = null;
     },
 
+    _haveClubhouse: function() {
```

Comment:
Nitpick: I'd prefer the method to be called `_hasClubhouse` because it sounds better and more clear than `this._haveClubhouse`. It'd be also more consistent with glib/gtk libs.
I am fine if you don't want to change it though.

Reply:
Fixed